compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -48,6 +50,9 @@ services:
     depends_on:
       - beacon
     command: --config-file=/config/validator.yaml
+    ports:
+      - 8081:8081/tcp # for web-ui    
```

Comment:
please add 127.0.0.1 to bind on on localhost and make it inacessible from outside

Reply:
got it

Reply:
`127.0.0.1` is added